### PR TITLE
LOG-3112: Add openshift.cluster_id to collection for fluentd

### PR DESCRIPTION
**Pull request by jcantrill**
### Description
This PR:
* Adds openshift.cluster_id to records collected by fluentd

### Links
* git push --set-upstream origin log3112

### CODE REVIEW

**Comment by jcantrill** [comment]
depends on https://github.com/ViaQ/logging-fluentd/pull/48

File: internal/generator/fluentd/pipeline_to_output_test.go [hunk 1]
```diff
@@ -102,9 +102,10 @@ var _ = Describe("Testing Config Generation", func() {
 <label @APP_TO_ES>
   # Add User Defined labels to the output record
   <filter **>
-    @type record_transformer
+    @type record_modifier
+    remove_keys _dummy_
     <record>
-      openshift { "labels": {"a":"b","c":"d"} }
+      _dummy_ ${record['openshift']={"labels"=>{}} unless record['openshift'];record['openshift']['labels'] = {"a"=>"b","c"=>"d"} }
```
Comment (alanconway) [inline internal/generator/fluentd/pipeline_to_output_test.go:108]: Why the _dummy_ field?

Comment (jcantrill) [reply internal/generator/fluentd/pipeline_to_output_test.go:108]: Its the recommendation to basically "script" functionality without writing a plugin... and then the field gets removed

Comment (alanconway) [reply internal/generator/fluentd/pipeline_to_output_test.go:108]: /lgtm

**Pull request merged**